frontend/src/app/credit-card/credit-card.component.css

```diff
@@ -0,0 +1,11 @@
+table{
+    width: 100%;
+}
+
+.mat-cell{
+    padding: 0px 8px 0px 0px;
+}
+
+.mat-header-cell{
+    padding: 0px 8px 0px 0px;
+}
```

Comment:
Any such CSS that will be used application-wide does not belong in individual component's CSS

This is a repetition of what we have in `commission.component.css`